From 7126ee563cdb6f3b2702dc6094c35e50397df5f6 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Mon, 3 Jun 2019 22:00:42 -0700
Subject: [PATCH 28/29] Revert "kernel: Correct CROSS_COMPILE_ARM32 toolchain"

This breaks build when building with upstream commit
ad15006cc78459d059af56729c4d9bed7c7fd860. That commit fixes LD being
used from outside the path, however androidkernel toolchains don't
have elfedit. Google specifically doesn't use androidkernel, so we
need to follow them on this.

This reverts commit c77bb3a5ba45699aeef45d3af8b8a0a59f0c6a14.

Change-Id: I79ac4ac47d3ba6c2abbc5ce40e56ed5c707295b4
---
 config/BoardConfigKernel.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/BoardConfigKernel.mk b/config/BoardConfigKernel.mk
index 8a0cf554..bc6f93d5 100644
--- a/config/BoardConfigKernel.mk
+++ b/config/BoardConfigKernel.mk
@@ -95,7 +95,7 @@ endif
 
 # Needed for CONFIG_COMPAT_VDSO, safe to set for all arm64 builds
 ifeq ($(KERNEL_ARCH),arm64)
-   KERNEL_CROSS_COMPILE += CROSS_COMPILE_ARM32="arm-linux-androidkernel-"
+   KERNEL_CROSS_COMPILE += CROSS_COMPILE_ARM32="arm-linux-androideabi-"
 endif
 
 # Clear this first to prevent accidental poisoning from env
-- 
2.17.1

